man/tpm2_quote.1.in

```diff
@@ -1,91 +0,0 @@
-.\" Copyright (c) 2016, Intel Corporation
-.\" All rights reserved.
-.\"
-.\" Redistribution and use in source and binary forms, with or without
-.\" modification, are permitted provided that the following conditions are met:
-.\"
-.\" 1. Redistributions of source code must retain the above copyright notice,
-.\" this list of conditions and the following disclaimer.
-.\"
-.\" 2. Redistributions in binary form must reproduce the above copyright notice,
-.\" this list of conditions and the following disclaimer in the documentation
-.\" and/or other materials provided with the distribution.
-.\"
-.\" 3. Neither the name of Intel Corporation nor the names of its contributors
-.\" may be used to endorse or promote products derived from this software without
-.\" specific prior written permission.
-.\"
-.\" THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
-.\" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
-.\" IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
-.\" ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
-.\" LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
-.\" CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
-.\" SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
-.\" INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
-.\" CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
-.\" ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
-.\" THE POSSIBILITY OF SUCH DAMAGE.
-.TH tpm2_quote 8 "DECEMBER 2016" Intel "tpm2-tools"
-.SH NAME
-tpm2_quote\ - Provide quote and signature for given list of PCRs in given algorithm/banks.
-.SH SYNOPSIS
-.B tpm2_quote[ COMMON OPTIONS ] [ TCTI OPTIONS ] [ \fB\-\-akHandle\fR|\fB\-\-akContext\fR|\fB\-\-akPassword\fR|\fB\-\-idList\fR|\fB\-\-algorithm\fR|\fB\-\-selList\fR|\fB\-\-outFile\fR|\fB\-\-passwdInHex\fR|\fB\-\-qualifyData\fR|\fB\-\-input-session-handle\fR|\fB ]
-.PP
-Provide quote and signature for given list of PCRs in given algorithm/banks.
-.SH DESCRIPTION
-.B tpm2_quote
-Provide quote and signature for given list of PCRs in given algorithm/banks.
-.SH OPTIONS
-.TP
-\fB\-k ,\-\-akHandle\fR
-Handle of existing AK
-.TP
-\fB\-c ,\-\-akContext\fR
-filename for the existing AK's context
-.TP
-\fB\-P ,\-\-akPassword\fR
-AK handle's Password.
-@PASSWORD_FORMAT_COMMON_INCLUDE@
-.TP
-\fB\-l ,\-\-idList\fR
```

Comment:
You missed this option in the new markdown man page.

Reply:
Done